Lib/axisregistry/data/roundness.textproto

```diff
@@ -0,0 +1,14 @@
+# ROND based on internal project
+tag: "ROND"
+display_name: "Roundness"
+min_value: 0
+default_value: 0
+max_value: 100
+precision: 0
+fallback {
+  name: "Default"
+  value: 0
+}
+fallback_only: false
+description:
+  "Adjust shapes from square to increasingly rounded forms."
```

Comment:
Softness has:

> Adjust letterforms from a sharp default (0) to become more and more soft and rounded (up to 100).

I'll make an edit to conform to this style of description.

Reply:
But we avoid using too specific terms like "letterforms" o "square," at least for broad concepts like Roundness. Part of our reasoning was if we added them, it could not apply to fonts that are either made not out of letterforms (like symbols), or that start from already slightly rounded forms (as Thomas's case)

Reply:
Note that in my case, we ALSO have completely sharp forms at the zero point on the axis. If we did not go in the Sharp direction, our default value would be non-zero.